### DISCUSSION TITLE
Looking to learn more from the monorepo...

### DISCUSSION THREAD

**Discussion by kevinsvibing** [dailydotdev/daily#879] [General]
Where can I checkout the code for `https://app.daily.dev/?utm_source=landing&utm_medium=cta&utm_campaign=landing_conversions`?

**Comment by idoshamun** [comment]
Hi @kgonzale,

Here you go: https://github.com/dailydotdev/apps

**Reply by kevinsvibing** [reply]
I see different packages, https://github.com/dailydotdev/apps/tree/main/packages 
which one should I be taking a look at?

**Reply by idoshamun** [reply]
They all contribute to our web application. But for app.daily.dev the main package is `webapp`
